src/plugins/data/public/plugin.ts

```diff
@@ -50,23 +52,35 @@ export class DataPublicPlugin implements Plugin<DataPublicPluginSetup, DataPubli
         uiSettings: core.uiSettings,
         storage,
       }),
+      indexPatterns: this.indexPatterns.setup(),
```

Comment:
This can be removed - no need to have it on the setup contract if it's empty

Reply:
Done